README.md

```diff
@@ -105,4 +105,20 @@ sequenceDiagram
     osw-database->>+osw-dataservice:Result
     osw-dataservice->>+Gateway:List of OSW
     Gateway->>+Client: OSW files list
+```
+
+
+## How to run integration test
+To run integration test you need a .env file which will be available on request.
+
```

Comment:
Please add table with Env variable dependency to run the integration test

Reply:
added